core-plugins/src/main/java/co/cask/hydrator/plugin/transform/PythonEvaluator.java

```diff
@@ -179,10 +180,17 @@ public void transform(StructuredRecord input, final Emitter<StructuredRecord> em
     } catch (PyException e) {
       // We put the stack trace as the exception message, because otherwise the information from PyException is lost.
       // PyException only exposes the actual cause (Python stack trace) if printStackTrace() is called on it.
-      throw new IllegalArgumentException("Could not transform input.\n" + ExceptionUtils.getStackTrace(e));
+      throw new IllegalArgumentException("Could not transform input.\n" + getStackTrace(e));
```

Comment:
BTW, why the stacktrace is included as part of the message instead of the cause of the IllegalArgumentException?

Reply:
from what I remember, PyException has funny behavior where it overrides printStackTrace and does special logic. @anwar6953 may remember more details.

Reply:
I see.

Reply:
I don't recall the details, but apparently, my comment above it mentions something similar to the reason Albert gave.